docs/reference/commands/node-tool.asciidoc

```diff
@@ -0,0 +1,329 @@
+the node on which it runs as follows:
+
+[source,txt]
+----
+Current node cluster state (term, version) pair is (4, 12)
+----
+
+If you have a choice of nodes on which to run this tool then you should choose
+one with a term that is as large as possible, and if there is more than one
+node with the same term then you should pick the one with the largest version.
+This identifies the node with the freshest cluster state, minimising the
+quantity of data that might be lost. For example, if the first node reports
+`(4, 12)` and a second node reports `(5, 3)`, then the second node is preferred
+since its term is larger.  However if the second node reports `(3, 17)` then
+the first node is preferred since its term is larger. If the second node
+reports `(4, 10)` then it has the same term as the first node, but has a
+smaller version, so the first node is preferred.
+
+[WARNING]
+Running this command can lead to arbitrary data loss. Only run this tool if you
+understand and accept the possible consequences and have exhausted all other
+possibilities for recovery of your cluster.
+
+The sequence of operations for using this tool are as follows:
+
+1. Make sure you have really lost access to at least half of the
+master-eligible nodes in the cluster, and they cannot be repaired or recovered
+by moving their data paths to healthy hardware.
+2. Stop **all** remaining nodes.
+3. Choose one of the remaining master-eligible nodes to become the new elected
+master as described above.
+4. On this node, run the `elasticsearch-node unsafe-bootstrap` command as shown
+below. Verify that the tool reported `Master node was successfully
+bootstrapped`.
+5. Start this node and verify that it is elected as the master node.
+6. Run the `elasticsearch-node detach-cluster` tool, described below, on every other
+node in the cluster.
+7. Start all other nodes and verify that each one joins the cluster.
+8. Investigate the data in the cluster to discover if any was lost during this
+process.
+
+[WARNING]
+When you run the tool it will make sure that the node that is being used to
+bootstrap the cluster is not running. It is important that all other
+master-eligible nodes are also stopped while this tool is running, but the tool
+does not check this.
+
+[NOTE]
+The message `Master node was successfully bootstrapped` does not mean that
+there has been no data loss, it just means that tool was able to complete its
+job.
+
+
+[float]
+==== Detaching nodes from their cluster
+
+It is unsafe for nodes to move between clusters, because different clusters
+have completely different cluster metadata. There is no way to safely merge the
+metadata from two clusters together.
+
+To protect against inadvertently joining the wrong cluster, each cluster
+creates a unique identifier, known as the _cluster UUID_, when it first starts
+up. Every node records the UUID of its cluster and will refuse to join a
+cluster with a different UUID in future.
```

Comment:
Actually, `unsafe-bootstrap` also generates new clusterUUID. However, we only talk about clusterUUID and in `unsafe-bootstrap` section we talk only about resetting voting configuration. I don't hold the opinion strong, but probably we can mention clusterUUID in `unsafe-bootstrap` section.

Reply:
I pushed 8e46c7dcf9416154c3110d56e56dfbe91aaaefd2 to emphasise it's a new cluster formed by `unsafe-bootstrap`, but without defining what this means in terms of the cluster UUID.